docs/api/lib-auditlog.adoc

```diff
@@ -14,15 +14,15 @@ Add the following into your `build.gradle` file:
 [source,groovy]
 ----
 dependencies {
-  include "com.enonic.xp:lib-audit:${xpVersion}"
+  include "com.enonic.xp:lib-auditlog:${xpVersion}"
 }
 ----
 
 In your JavaScript controller, add a require statement:
 
 [source,js]
 ----
-const authLib = require('/lib/xp/audit');
```

Comment:
Did it really say authLib? 🙈 I suspect copy paste

Reply:
auth and audit is pretty close. Easy mistake :smile: